marklogic-data-hub/src/test/resources/entity-reference-model/flows/ingestToFinal.flow.json

```diff
@@ -37,6 +37,12 @@
         "targetDatabase": "data-hub-STAGING",
         "permissions": "data-hub-common,read,data-hub-common,update"
       }
+    },
```

Comment:
Why not use the existing ingestion / mapping steps in this flow? I ask because this project gets loaded by a lot of tests, so I'd like to only add a new artifact to it if we really need it.

Reply:
Added new steps to verify if the step name is returned or not. Otherwise step-defintion name is returned when the existing steps are used.